fetch.bs

```diff
@@ -1870,6 +1881,17 @@ is a <a>filtered response</a> whose
  <li><p>Return <var>newResponse</var>.
 </ol>
 
+<p>A <dfn id=concept-fresh-response>fresh response</dfn> is a <a for=/>response</a> whose age is
+within its <a href=https://tools.ietf.org/html/rfc7234#section-4.2.1>freshness lifetime</a>.
+
+<p>A <dfn id=concept-stale-while-revalidate-response>stale-while-revalidate response</dfn> is a
+<a for=/>response</a> that is not a <a>fresh response</a>, and its current age is within
```

Comment:
Is "current age" different from age?

Reply:
yes, according the rfc.  there is actually nothing like pure "age".  but there are apparent, cached, corrected received, corrected initial and current age.  current age is based on all of the others and is the one to compare with freshness lifetime.

Reply:
I linked it to the definition of current age.